LibVLCSharp.Gtk/VideoView.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System;
+using System.Runtime.InteropServices;
+using Gdk;
+using Gtk;
+using LibVLCSharp.Shared;
+using Object = Gtk.Object;
+
+namespace LibVLCSharp.Gtk
+{
+    public class VideoView : DrawingArea, IVideoView
+    {
+        private MediaPlayer _mediaPlayer;
+
+        public VideoView(MediaPlayer mediaPlayer = null)
```

Comment:
The `VideoView` ctor on the other platforms does not have this optional parameters. Either we remove it here or we add it in the other `VideoView` implementations.

Reply:
I found it easier to use in the Gtk# sample. If you want to use that in the WPF control, it's hard to pass a constructor parameter.

I'm ok with both ways, just tell me